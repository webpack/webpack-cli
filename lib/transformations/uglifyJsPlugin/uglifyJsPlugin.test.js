const defineTest = require('../defineTest');

defineTest(__dirname, 'uglifyJsPlugin', 'uglifyJsPlugin-0');
defineTest(__dirname, 'uglifyJsPlugin', 'uglifyJsPlugin-1');
defineTest(__dirname, 'uglifyJsPlugin', 'uglifyJsPlugin-2');
