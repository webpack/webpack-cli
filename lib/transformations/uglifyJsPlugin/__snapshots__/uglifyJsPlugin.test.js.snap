// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-0" data 1`] = `
"module.exports = {
    plugins: [
        new webpack.optimize.UglifyJsPlugin({
            sourceMap: true
        })
    ]
}
"
`;

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-1" data 1`] = `
"module.exports = {
    devtool: \\"source-map\\",
    plugins: [
        new webpack.optimize.UglifyJsPlugin({
            sourceMap: true
        })
    ]
}
"
`;

exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-2" data 1`] = `
"module.exports = {
    devtool: \\"cheap-source-map\\",
    plugins: [
        new webpack.optimize.UglifyJsPlugin({
            compress: {},
            sourceMap: true
        })
    ]
}
"
`;
